src/equations/linearized_euler_2d.jl

```diff
@@ -0,0 +1,184 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+@doc raw"""
+    LinearizedEulerEquations2D(rho, lambda)
+    
+The linearized Euler equations
+```math
+\partial t
+\begin{pmatrix}
+  v_1 \\ v_2 \\ p
+\end{pmatrix}
++
+\partial x
+\begin{pmatrix}
+\frac{1}{\rho} p \\ 0 \\ \lambda v1
+\end{pmatrix}
++
+\partial y
+\begin{pmatrix}
+0 \\ \frac{1}{\rho} p \\ \lambda v2
+\end{pmatrix}
+=
+\begin{pmatrix}
+0 \\ 0 \\ 0
+\end{pmatrix}
+```
+for an ideal acoustic wave with constant density ``\rho`` and second lame constant ``\lambda``.
```

Comment:
```suggestion
for an ideal acoustic wave with constant density ``\rho`` and second Lamé constant ``\lambda``.
```
I believe this is the correct spelling